tests/autoyast/installation.pm

```diff
@@ -106,7 +106,7 @@ sub verify_timeout_and_check_screen {
 sub run {
     my ($self) = @_;
 
-    test_ayp_url unless get_var('VIRT_AUTOTEST') and is_tumbleweed;
+    test_ayp_url unless get_var('IPXE_STATIC');
```

Comment:
From name `test_ayp_url`, it seems this function is good to have for those using autoyast. What's the purpose of this unless condition? Exclude virt tests or really those with ipxe_static?

Reply:
`test_ayp_url` is to check the dynamic autoyast profile url accessible. but for tests with static ipxe menu, the url is static rathar than gererated based on job ID, so `test_ayp_url` is not applied to this case. Replace the settings because `IPXE_STATIC` was a new setting added recently.

Reply:
Make sense to me, thanks.

Reply:
Julie this function `test_ayp_url` checks dynamic AutoYast profile URLs and now will run for 'VIRT_AUTOTEST' and tumbleweed.
And now not run for 'IPXE_STATIC'?

Reply:
we have `IPXE_STATIC` for `VIRT_AUTOTEST` TW tests. so there is no change actually.